### PR TITLE
Remove providerId from Token

### PR DESCRIPTION
**Pull request by fredx30**
The database model appears to not contain any key for `providerId`. This PR removes that key from the object that models that table.

### CODE REVIEW

File: pkg/wharfapi/token.go [hunk 1]
```diff
@@ -11,7 +11,6 @@ type Token struct {
 	TokenID    uint   `json:"tokenId"`
 	Token      string `json:"token"`
 	UserName   string `json:"userName"`
-	ProviderID uint   `json:"providerId"`
```
Comment (jilleJr) [inline pkg/wharfapi/token.go:14]: Oh yea this was removed from the database a while ago as it created a circular dependency between the token and the provider.

Do you mind also updating the CHANGELOG.md?

**Review by jilleJr — CHANGES_REQUESTED** [review]

File: pkg/wharfapi/token.go (hunk 1 above)
Comment (jilleJr) [reply pkg/wharfapi/token.go:14]: Nope. You probably forgot to push

Comment (fredx30) [reply pkg/wharfapi/token.go:14]: Trial 2 on this up now. I verified i can see the Changelog changes on this PR.

**Comment by fredx30** [comment]
Conflicts resolved.

**Review by jilleJr — APPROVED** [review]
LGTM!

**Review by Pikabanga — APPROVED** [review]

**Pull request merged**